src/sql/src/plan/statement/ddl.rs

```diff
@@ -2354,6 +2372,7 @@ pub fn plan_drop_objects(
         | ObjectType::Sink
         | ObjectType::Type => plan_drop_items(scx, object_type, if_exists, names, cascade),
         ObjectType::Role => plan_drop_role(scx, if_exists, names),
+        ObjectType::Secret => unreachable!("DROP SECRET"),
```

Comment:
You'll want `bail_unreachable!` here, I think, to avoid panicking!

Reply:
`bail_unsupported`?

Reply:
I will open a follow up PR for DROP SECRET and make this change

Reply:
Sorry, yeah, `bail_unsupported`.